server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -71,6 +71,64 @@ public void toPlainTextShouldUpperCaseLabels() {
     }
 
     @Test
+    public void toPlainTextShouldHandleListsWell() {
+        String html = "<ul>Here is my awesome list:" +
+            "  <li>JMAP</li>" +
+            "  <li>IMAP</li>" +
+            "</ul>" +
+            "<p>Followed with some text</p>" +
+            "<p>And some other text</p>";
+        String expectedPlainText = "Here is my awesome list:  \n" +
+            " - JMAP  \n" +
+            " - IMAP\n" +
+            "\n" +
+            "Followed with some text\n" +
+            "\n" +
+            "And some other text\n" +
+            "\n";
+        assertThat(textExtractor.toPlainText(html)).isEqualTo(expectedPlainText);
+    }
+
+    @Test
+    public void tableShouldBeWellHandled() {
+        String html = " <table style=\"width:100%\">\n" +
```

Comment:
No, the HTML can contain anything that we want but this test mean "\n" should be extracted to "\n" IMO. It's just my thought.